guides/common/modules/con_configuring-single-sign-on-and-two-factor-authentication-with-keycloak-wildfly-in-project.adoc

```diff
@@ -1,5 +1,5 @@
-[id="configuring-keycloak-wildfly-authentication-for-project_{context}"]
-= Configuring {keycloak-wildfly} authentication for {Project}
+[id="configuring-single-sign-on-and-two-factor-authentication-with-keycloak-wildfly-in-project_{context}"]
+= Configuring single sign-on and two-factor authentication with {keycloak-wildfly} in {Project}
```

Comment:
The heading looks too long and that may be pointing in favor of splitting this module into two. If they would be almost the same, it can be a single parametrized module with differences as attributes.